### ISSUE TITLE
Menu items are not navigatable when returning to same menu item

### ISSUE THREAD

**Issue by andreievg** [openmsupply/application-manager-web-app#917]
Try navigating to something from menu item, then to dashboard, then back to the same item.

I think need to reset selection (as it's not being registered), or maybe some other option in dropdown

**Comment by CarlosNZ** [comment]
Yeah, I noticed this. I think each seperate Dropdown in the menu bar is remembering its own state, and only does something "onChange", so even though the page has changed, the actual selected value in the previous dropdown hasn't changed. Should be an easy fix, I'll push one out this morn.
